packaging/helm-charts/helm3/strimzi-kafka-operator/templates/060-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -118,6 +118,10 @@ spec:
             - name: STRIMZI_NETWORK_POLICY_GENERATION
               value: {{ .Values.generateNetworkPolicy }}
             {{- end }}
+            {{- if ne (.Values.connectBuildTimeoutMs | quote) "300000" }}
+            - name: STRIMZI_CONNECT_BUILD_TIMEOUT_MS
+              value: {{ .Values.connectBuildTimeoutMs | quote }}
+            {{- end }}
```

Comment:
It looks like this modifies the `060-Deployment-strimzi-cluster-operator.yaml` file and that fails the build. You should be able to debug it using `make helm_install`.

Reply:
Yeah, Thanks @scholzj. I am trying to debug it now and find out the root cause